apps/gp2-frontend/src/AuthenticatedApp.tsx

```diff
@@ -43,13 +46,25 @@ const AuthenticatedApp: FC<Record<string, never>> = () => {
 
   useEffect(() => {
     // order by the likelyhood of user navigating there
-    loadDashboard().then(loadUsers).then(loadWorkingGroups).then(loadProjects);
-  }, []);
+    user?.onboarded
+      ? loadDashboard()
+          .then(loadUsers)
+          .then(loadWorkingGroups)
+          .then(loadProjects)
+      : loadOnboarding()
+          .then(loadDashboard)
+          .then(loadUsers)
+          .then(loadWorkingGroups)
+          .then(loadProjects);
```

Comment:
Do we need to do the lazy loading of all the other pages when the user isn't onboarded. They can't see them until they are, and then this will re-render?